mysqld_exporter.go

```diff
@@ -52,6 +63,56 @@ func NewMySQLExporter(dsn string) *Exporter {
 			Help:      "The last scrape error status.",
 		}),
 		metrics: map[string]prometheus.Gauge{},
+		commands: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "commands_total",
+			Help:      "Number of executed mysql commands.",
+		}, []string{"command"}),
+		connection_errors: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "connection_errors_total",
+			Help:      "Number of mysql connection errors.",
+		}, []string{"error"}),
+		handlers: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "handlers_total",
+			Help:      "Number of executed mysql handlers.",
+		}, []string{"handler"}),
+		innodb_buffer_pool: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_buffer_pool_total",
```

Comment:
What's the unit on this one? Bytes? Pages? Or are there varying units in this metric (if so, it should probably not be a single metric)?